src/ert/job_queue/driver.py

```diff
@@ -101,65 +106,130 @@ async def kill(self, realization: "RealizationState") -> None:
         realization.verify_kill()
 
 
+LSF_STATUSES = [
+    "PEND",
+    "SSUSP",
+    "PSUSP",
+    "USUSP",
+    "RUN",
+    "EXIT",
+    "ZOMBI",
+    "DONE",
+    "PDONE",
+    "UNKWN",
+]
+
+
 class LSFDriver(Driver):
     def __init__(self, queue_options: Optional[List[Tuple[str, str]]]):
         super().__init__(queue_options)
 
         self._realstate_to_lsfid: Dict["RealizationState", str] = {}
         self._lsfid_to_realstate: Dict[str, "RealizationState"] = {}
+        self._max_attempt = 100
+        self._MAX_ERROR_COUNT = 100
+        self._statuses = {}
         self._submit_processes: Dict[
             "RealizationState", "asyncio.subprocess.Process"
         ] = {}
-
+        self._SLEEP_PERIOD = 3
```

Comment:
do we want upper case? I would do lower..

Since this is a class variable, can you extend it to make it more explicit what this sleep is to be used for?